### ISSUE TITLE
Feature - PipeCAD.Core - Add version number of PipeCAD

### ISSUE THREAD

**Issue by rompik** [eryar/PipeCAD#13]
Please can you can add to PipeCad opportunity to get version of PipeCAD inside Python (for ex. possible code in python):
```python
self.labelSoftwareVersion = QLabel( PipeCAD.Core.Version() )
```

**Comment by eryar** [comment]
I have create a project and invited you as admin.

**Comment by eryar** [comment]
Fixed PipeCAD-1.0.4
